xonsh/platform.py

```diff
@@ -240,7 +246,12 @@ def scandir(path):
                      'usr\\share\\bash-completion\\completions'),
         os.path.join(GIT_FOR_WINDOWS_PATH,
                      'mingw64\\share\\git\\completion\\git-completion.bash'))
-
+    PATH_DEFAULT = tuple(winreg.QueryValueEx(winreg.OpenKey(
+            winreg.HKEY_LOCAL_MACHINE,
+            r'SYSTEM\CurrentControlSet\Control\Session Manager\Environment'),
+        'Path')[0].split(os.pathsep))
```

Comment:
I haven't followed all the discussions on gitter, but why is this necessary? Don't we still inherit the environment from `os.envrion` on startup? What is the situation where it would use this default value?